libraries/previousAuctionInfo/previousAuctionInfo.js

```diff
@@ -0,0 +1,130 @@
+import {on as onEvent} from '../../src/events.js';
+import { EVENTS } from '../../src/constants.js';
+
+export let previousAuctionInfoEnabled = false;
+let enabledBidders = [];
+export let winningBidsMap = {};
+
+export const resetPreviousAuctionInfo = () => {
+  previousAuctionInfoEnabled = false;
+  enabledBidders = [];
+  winningBidsMap = {};
+};
+
+export const enablePreviousAuctionInfo = (config, cb = initHandlers) => {
```

Comment:
It's unclear to me how this is intended to be used. 

If it's an optional module chosen by the publisher, this should be in the `modules` folder and listen for changes to `setConfig`.

Alternatively we could let individual bidders opt in, in which case the only parameter needed should be the bidder code, and this should allow for being called multiple times with different bidders.